tests/components/point/test_config_flow.py

```diff
@@ -65,7 +65,7 @@ async def test_abort_if_already_setup(hass: HomeAssistant) -> None:
     assert result["reason"] == "already_setup"
 
     with patch.object(hass.config_entries, "async_entries", return_value=[{}]):
-        result = await flow.async_step_import()
+        result = await flow.async_step_import(None)
```

Comment:
Note: I think the tests should be refactored, as they currently bypass `hass.config_entries.flow.async_init` and thus do not reflect the reality.
However, I think that should be done by the code-owner in a follow-up PR.